redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+                    "Reacting with \N{WHITE HEAVY CHECK MARK} will continue "
+                    "the mute with overwrites and stop this message from appearing again, "
+                    "Reacting with \N{NEGATIVE SQUARED CROSS MARK} will end the mute attempt."
+                )
+            else:
+                msg += _(
+                    "Saying `yes` will continue "
+                    "the mute with overwrites and stop this message from appearing again, "
+                    "saying `no` will end the mute attempt."
+                )
+            query: discord.Message = await ctx.send(msg)
+            if can_react:
+                # noinspection PyAsyncCall
+                start_adding_reactions(query, ReactionPredicate.YES_OR_NO_EMOJIS)
+                pred = ReactionPredicate.yes_or_no(query, ctx.author)
+                event = "reaction_add"
+            else:
+                pred = MessagePredicate.yes_or_no(ctx)
+                event = "message"
+            try:
+                await ctx.bot.wait_for(event, check=pred, timeout=30)
+            except asyncio.TimeoutError:
+                await query.delete()
+                return False
+
+            if not pred.result:
+                if can_react:
+                    await query.delete()
+                else:
+                    await ctx.send(_("OK then."))
+
+                return False
+            else:
+                if can_react:
+                    with contextlib.suppress(discord.Forbidden):
+                        await query.clear_reactions()
+                await self.config.guild(ctx.guild).sent_instructions.set(True)
+                return True
+
+    @commands.command()
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def activemutes(self, ctx: commands.Context):
+        """
+        Displays active mutes on this server.
+        """
+
+        msg = ""
+        to_del = []
+        if ctx.guild.id in self._server_mutes:
+            mutes_data = self._server_mutes[ctx.guild.id]
+            to_rem = []
+            if mutes_data:
+
+                msg += _("__Server Mutes__\n")
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        to_rem.append(user_id)
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member}").format(member=user_str)
```

Comment:
There's nothing to translate here + you missed a space:
```suggestion
                    msg += f"{user_str} "
```